website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md

```diff
@@ -2,159 +2,59 @@
-
-:::info
-Bind the query’s response to the Table using JavaScript in the Table Data Property **`{{list_files.data}}`**.
-:::
-
-Now your table should list all the files present in your S3 bucket.
-
-You can use an image widget to display images listed in the table widget. You can follow this [guide](/connect-data/how-to-guides/how-to-upload-to-s3)to learn more.
-
-#### Download Files
-
-* Open the image property pane.
-* Click the **JS** button next to the **`onClick`** event and write the following JavaScript query:
-
-```
-{{download(atob(Fetch_image.data.fileData),'Testname','image/png')}}
-```
-
-Now, your image is downloaded when you click on the image widget.
-
-:::info
-You can check this [Guide](/connect-data/how-to-guides/how-to-upload-to-s3)to learn more about Upload/Download Files from S3.
-:::
-
-With Appsmith S3 integration, it is possible to create apps that seamlessly connect with the S3 database and provide additional flexibility for updating and analyzing data.
-
+# Upload Images Using Camera
+Appsmith lets you capture images and videos from your applications and share the data for further use.
+For more information, see [Camera](https://docs.appsmith.com/reference/widgets/camera).
+
+This page provides steps to upload images using the Camera widget to a datasource and uses the S3 datasource as an example.
+You will learn how to:
+- [Create an S3 datasource and connect your app.](#create-an-s3-datasource)
+- [Upload an image to S3 using the Camera widget.](#upload-an-image-using-the-camera-widget)
+
+## Prerequisites
+- To upload your data to Amazon S3, you must first [create an Amazon S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html) in one of the AWS Regions.
+
+## Create an S3 Datasource
+Amazon Simple Storage Service (Amazon S3) is an object storage service that lets you store, manage, and protect any amount of data. Using Amazon S3, you can manage and organize your data based on your business, organizational, and compliance needs.
+
+To add an S3 datasource to your workspace, follow these steps:
+1. In your Appsmith workspace, select your app and navigate to the **Explorer** tab in the page.
+2. In **Datasources**, click **+ Add a new datasource**. 
+3. Select **S3**.
+4. Once your have created the S3 datasource, follow these [steps](/connect-data/reference/querying-amazon-s3#connection-settings) to connect your app to the S3 datasource.
+
+## Upload an image using the Camera widget
+To upload an image or a video to Amazon S3, follow these steps:
+1. Drag and drop the [Camera widget](/reference/widgets/camera) onto the canvas.
+2. In **Queries/JS**, click the **+ Add a new query/JS Object**.
+3. Select your S3 datasource.
+4. In **Commands**, select **Create a new file**.
+5. Configure the following parameters for the new file:
+   * **Bucket Name**: Name of the Amazon S3 bucket.
+   * **File Path**: Path of the location you want to store the file. For example, `images/any`.
```

Comment:
Could you please add an example or rewrite it with respect to the path available in the S3 bucket?

Reply:
Modified the example to "images/camera_uploads". This is the path I have set for my testing.